portal/__init__.py

```diff
@@ -65,6 +65,10 @@ def create_app(test_config=None):
     from . import roster
     app.register_blueprint(roster.bp)
 
+    from . import schedule
+    app.register_blueprint(schedule.bp)
+
+
```

Comment:
Hey, So the schedule should not have its own view. Lets put it in a file which we will call something like student_views.py. Which will hold students viewing courses, grades, and assignments.